src/cmap/connection.ts

```diff
@@ -661,6 +661,7 @@ function write(
     command: !!options.command,
 
     // for BSON parsing
+    useBigInt64: typeof options.useBigInt64 === 'boolean' ? options.useBigInt64 : false,
```

Comment:
When will this ever _not_ be set? 

`resolveBSONOptions` will ensure that this is always set to a value

Reply:
Sure, that works